README.md

```diff
@@ -30,14 +30,19 @@ npm run build
 
 ### Publishing Packages
 
-#### With 2FA on your NPM account
+#### Setting up your npm account
 
-```
-NPM_CONFIG_OTP=<2FA_Code> npm run publish
-```
+- Create an npm account with your bbc email address. https://www.npmjs.com
+- In your npm profile settings, set up Two Factor Authentication. Enable it for authorization and publishing
+- Back up your recovery codes for your account
+- Join the BBC npm org by following the process here: https://github.com/bbc/npm When you're added to the org and signed in, you should be able to see all public and private `@bbc` packages here: https://www.npmjs.com/settings/bbc/packages
 
-### Without 2FA
+#### Publishing a package
 
-```
-npm run publish
-```
+- In your terminal, run `npm login` and follow the steps for authentication
+- `npm publish packages/<PATH_TO_PACKAGE>`
```

Comment:
Repeating my comment on the issue:

npm publish packages/<PATH_TO_PACKAGE>
e.g. npm publish packages/components/psammead-paragraph
I think this line should also include the --otp=<YOUR_2FA_CODE>, as I got an error & re-prompted for it. 😄

Reply:
```suggestion
- `npm publish packages/<PATH_TO_PACKAGE> --otp=<YOUR_2FA_CODE>`
```

Reply:
Thanks!